pyntc/devices/asa_device.py

```diff
@@ -41,6 +40,16 @@ class ASADevice(BaseDevice):
     active_redundancy_states = {None, "active"}
 
     def __init__(self, host: str, username: str, password: str, secret="", port=22, **kwargs):
+        """
+        PyNTC Device implemenation for Cisco ASA
```

Comment:
Maybe make this a bit more accurate, change to: `PyNTC Device constructor for Cisco ASA`

Reply:
Good idea. Updated.